termqt/terminal_widget.py

```diff
@@ -9,7 +9,7 @@
 from .terminal_buffer import TerminalBuffer, DEFAULT_BG_COLOR, \
     DEFAULT_FG_COLOR, ControlChar
 
-PADDING = 4
+PADDING = 2
```

Comment:
Is this change motivated by the use of font?

Reply:
This is just testing that I forgot to revert, sorry. Will revert it.